torch/csrc/jit/script/module.cpp

```diff
@@ -12,6 +12,35 @@ namespace torch {
 namespace jit {
 namespace script {
 
+static ModulePtr create_module_object(c10::QualifiedName class_name) {
+  auto cu = std::make_shared<CompilationUnit>();
+  auto cls = ClassType::create(std::move(class_name), cu, /*is_module=*/true);
+  return c10::ivalue::Object::create(
+      c10::StrongTypePtr(std::move(cu), std::move(cls)), 0);
+}
+
+Module::Module(c10::QualifiedName class_name)
+    : module_value_(create_module_object(std::move(class_name))) {}
+
+Module::Module(
+    c10::QualifiedName class_name,
+    std::shared_ptr<CompilationUnit> cu) {
+  auto cls = ClassType::create(
```

Comment:
Shouldn't this be a call to create_module_object as well? Only difference is the cu